frontend/src/utils/StatusUtils.ts

```diff
@@ -0,0 +1,16 @@
+// Change status mysql enum to title case
+function convertStatusTitleCase(status: string) {
+  const statusEnum = status.substring(30);
```

Comment:
What's the point of this line?

Reply:
status actually comes from gql as `STORYTRANSLATIONCONTENT.DEFAULT` etc, so it needs some trimming

Reply:
nitpick, but would it be more clear to use a "replace STORYTRANSLATIONCONTENT. substring"?